### ISSUE TITLE
Bayesian logistic regression.

### ISSUE THREAD

**Issue by janderuiter** [jasp-stats/jasp-desktop#2119]
Could we have a Bayesian version of logistic regression as well? That would be SO cool! Thanks in advance for considering. 

JP

**Comment by oreojo** [comment]
I also echo this request - a bayesian logistic regression would be good!

**Comment by oreojo** [comment]
I may have answered this myself - can't you just convert the binary variable to one it thinks is linear? i.e. literally just change the drop-down 'nominal' to 'scale' on your categorical outcome measure? The size of the BF10's that I get back seem to make sense when I do this, but it might be fundamentally wrong?

**Comment by janderuiter** [comment]
No, I don't think this is the same analysis. For one thing, there's no
logistic function involved.
